src/go/gomod/gomod.go

```diff
@@ -0,0 +1,37 @@
+package gomod
+
+import (
+	"bufio"
+	"log"
+	"os"
+	"strings"
+)
+
+type GoMod struct {
+	GoVersion string
+}
+
+// TODO: Replace with https://pkg.go.dev/golang.org/x/mod/modfile when it is available
+func (g *GoMod) Load(path string) {
```

Comment:
This function should return an error. Also, every function used inside that may generate an error should be returned with a clear error message, do not use `log.fatal`.

Something like this:

```go
func (g *GoMod) Load(path string) error {
	f, err := os.Open(path)
	if err != nil {
		return fmt.Errorf("failed to open go.mod: %w", err)
	}
	.
	.
	.
	return nil
}
```

Reply:
Ok, thanks for the help!